tests/generator_test.php

```diff
@@ -141,7 +142,7 @@ public function test_create_session() {
         $this->assertIsArray($session2->sessiondates);
         $this->assertCount(1, $session2->sessiondates);
         $this->assertSame($session2->id, $session2->sessiondates[0]->sessionid);
-        $this->assertSame((string)($now - 3 * DAYSECS), $session2->sessiondates[0]->timestart);
-        $this->assertSame((string)($now - 2 * DAYSECS), $session2->sessiondates[0]->timefinish);
+        $this->assertSame((string) ($now - 3 * DAYSECS), $session2->sessiondates[0]->timestart);
+        $this->assertSame((string) ($now - 2 * DAYSECS), $session2->sessiondates[0]->timefinish);
```

Comment:
There were conflicts here, mainly because 403 has more assertions after this bit.